<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Updatedetails</class>
 <widget class="QDialog" name="Updatedetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(204, 168, 233);
color: rgb(245, 235, 235);
</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="update_dvc">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>311</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <property name="verticalSpacing">
       <number>25</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="horizontalSpacing">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(9,9,9)
</string>
          </property>
          <property name="text">
           <string>Device Name :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="dvc_name_textedit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(195, 190, 240);
    color: rgb(9, 9, 9);
    /*padding: 12px 60px; /* Adjust padding as needed */
    border-radius: 8px;
    border: 1px solid black;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout_4">
        <property name="horizontalSpacing">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(9,9,9)
</string>
          </property>
          <property name="text">
           <string>Serial No :      </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="sr_no_textedit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(195, 190, 240);
    color: rgb(9, 9, 9);
    /*padding: 12px 60px; /* Adjust padding as needed */
    border-radius: 8px;
    border: 1px solid black;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 190, 240);
color: rgb(9,9,9);
padding : 15px 20px;
border-radius:8px;
border: 1px solid black;


</string>
        </property>
        <property name="text">
         <string>Add Device</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="update_user">
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>311</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_6">
      <property name="labelAlignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="verticalSpacing">
       <number>25</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout_7">
        <property name="horizontalSpacing">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(9,9,9)
</string>
          </property>
          <property name="text">
           <string>User Name:  </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="user_name_textedit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(195, 190, 240);
    color: rgb(9, 9, 9);
  /*  padding: 12px 60px; /* Adjust padding as needed */
    border-radius: 8px;
    border: 1px solid black;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout_8">
        <property name="horizontalSpacing">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(9,9,9)
</string>
          </property>
          <property name="text">
           <string>Location:    </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="Location_textedit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(195, 190, 240);
    color: rgb(9, 9, 9);
   /* padding: 12px 60px; /* Adjust padding as needed */
    border-radius: 8px;
    border: 1px solid black;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="update_user_button">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(195, 190, 240);
color: rgb(9,9,9);
padding : 15px 20px;
border-radius:8px;
border: 1px solid black;


</string>
        </property>
        <property name="text">
         <string>Add User</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>363</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>100</number>
    </property>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(9,9,9)</string>
      </property>
      <property name="text">
       <string>Update Device</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(9,9,9)</string>
      </property>
      <property name="text">
       <string>Update User</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
